src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/resources/stylesheet.css

```diff
@@ -844,6 +844,15 @@ span#page-search-link {
     margin: -100px 0 0 100px;
     z-index: 1;
 }
+.sealed-graph span {
+    display:none;
+    position:absolute;
+}
+.sealed-graph:hover span {
+    display:block;
+    margin: -100px 0 0 100px;
+    z-index: 1;
+}
```

Comment:
These lines could be merged with the preceding lines, using the `,` syntax

Reply:
```

.module-graph span, .sealed-graph span {
    display:none;
    position:absolute;
}
.module-graph:hover span, .sealed-graph:hover span {
    display:block;
     margin: -100px 0 0 100px;
     z-index: 1;
 }
```

Reply:
Fixed. Thank you for helping me with the CSS syntax. I have verified that it looks good for both existing module graphs and the new sealed graphs.